ext/intl/calendar/calendar_methods.cpp

```diff
@@ -1023,8 +1020,8 @@ U_CFUNC PHP_FUNCTION(intlcal_from_date_time)
 	calendar_object_create(return_value, cal);
 
 error:
-	if (zv_datetime && zv_datetime != zv_arg) {
-		zval_ptr_dtor(zv_datetime);
+	if (date_obj && date_str) {
```

Comment:
How can date_obj be NULL here?

Reply:
There's no way -- `date_obj` is dropped from the condition.